<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-03T17:39:23 IST" name="Default suite" finished-at="2024-04-03T17:39:24 IST" duration-ms="1410">
    <groups>
    </groups>
    <test started-at="2024-04-03T17:39:23 IST" name="Default test" finished-at="2024-04-03T17:39:24 IST" duration-ms="1410">
      <class name="storeProcedureTesting.SpTesting">
        <test-method is-config="true" signature="setup()[pri:0, instance:storeProcedureTesting.SpTesting@71a8adcf]" started-at="2024-04-03T17:39:23 IST" name="setup" finished-at="2024-04-03T17:39:24 IST" duration-ms="953" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="test_storeProceduresExists()[pri:1, instance:storeProcedureTesting.SpTesting@71a8adcf]" started-at="2024-04-03T17:39:24 IST" name="test_storeProceduresExists" finished-at="2024-04-03T17:39:24 IST" duration-ms="115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_storeProceduresExists -->
        <test-method signature="test_selectAllCustomers()[pri:2, instance:storeProcedureTesting.SpTesting@71a8adcf]" started-at="2024-04-03T17:39:24 IST" name="test_selectAllCustomers" finished-at="2024-04-03T17:39:24 IST" duration-ms="121" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_selectAllCustomers -->
        <test-method signature="test_SelectAllCustomersByCityAndPin()[pri:3, instance:storeProcedureTesting.SpTesting@71a8adcf]" started-at="2024-04-03T17:39:24 IST" name="test_SelectAllCustomersByCityAndPin" finished-at="2024-04-03T17:39:24 IST" duration-ms="21" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_SelectAllCustomersByCityAndPin -->
        <test-method signature="test_get_order_by_cust()[pri:4, instance:storeProcedureTesting.SpTesting@71a8adcf]" started-at="2024-04-03T17:39:24 IST" name="test_get_order_by_cust" finished-at="2024-04-03T17:39:24 IST" duration-ms="40" status="FAIL">
          <exception class="java.sql.SQLSyntaxErrorException">
            <message>
              <![CDATA[You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '()' at line 1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '()' at line 1
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:916)
at com.mysql.cj.jdbc.ClientPreparedStatement.execute(ClientPreparedStatement.java:354)
at com.mysql.cj.jdbc.CallableStatement.execute(CallableStatement.java:847)
at storeProcedureTesting.SpTesting.test_get_order_by_cust(SpTesting.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.sql.SQLSyntaxErrorException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_get_order_by_cust -->
        <test-method signature="test_SetCustomerShipping()[pri:5, instance:storeProcedureTesting.SpTesting@71a8adcf]" started-at="2024-04-03T17:39:24 IST" name="test_SetCustomerShipping" finished-at="2024-04-03T17:39:24 IST" duration-ms="15" status="FAIL">
          <exception class="java.sql.SQLSyntaxErrorException">
            <message>
              <![CDATA[You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '()' at line 1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '()' at line 1
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:916)
at com.mysql.cj.jdbc.ClientPreparedStatement.execute(ClientPreparedStatement.java:354)
at com.mysql.cj.jdbc.CallableStatement.execute(CallableStatement.java:847)
at storeProcedureTesting.SpTesting.test_SetCustomerShipping(SpTesting.java:143)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.sql.SQLSyntaxErrorException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_SetCustomerShipping -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:storeProcedureTesting.SpTesting@71a8adcf]" started-at="2024-04-03T17:39:24 IST" name="tearDown" finished-at="2024-04-03T17:39:24 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- storeProcedureTesting.SpTesting -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
